internal/cmd/asyncapi/command_export.go

```diff
@@ -333,6 +333,11 @@ func (c *command) getClusterDetails(details *accountDetails, flags *flags) error
 	if err != nil {
 		return fmt.Errorf(`failed to find Kafka cluster: %v`, err)
 	}
+
+	if err := clusterConfig.DecryptAPIKeys(); err != nil {
```

Comment:
```suggestion
	if err := clusterConfig.DecryptApiKeys(); err != nil {
```

Reply:
Did we consider only decrypting the API keys we use?

Reply:
i think here multiple keys might be required depending on the conditions in following lines